pkg/processor/runtime/nodejs/test/outputter/outputter.js

```diff
@@ -15,60 +15,57 @@ limitations under the License.
 */
 
 // We use non default file name and handler name to test configuration as well
-exports.testHandler = function(context, event) {
-    if (event.method != 'POST') {
-        context.callback(event.method);
-	return;
+exports.testHandler = function (context, event) {
+    if (event.method !== 'POST') {
+        context.callback(event.method)
+        return
     }
 
-    var body = event.body.toString();
-
-    switch (body) {
-	case 'return_string':
-	    context.callback('a string');
-	    return;
-	case 'return_status_and_string':
-	    context.callback([201, 'a string after status']);
-	    return;
-	case 'return_list':
-	    context.callback([{a: 1}, {b: 2}]);
-	    return;
-	case 'return_status_and_dict':
-	    context.callback([201, {a: 'dict after status', b: 'foo'}]);
-	    return;
-	case 'log':
-	    context.logger.debug('Debug message');
-	    context.logger.info('Info message');
-	    context.logger.warn('Warn message');
-	    context.logger.error('Error message');
-	    context.callback([201, 'returned logs']);
-	    return;
-	case 'log_with':
-	    context.logger.errorWith('Error message', {source: 'rabbit', weight: 7});
-	    context.callback([201, 'returned logs with']);
-	    return;
-	case 'return_response':
-	    headers = event.headers;
-	    headers['h1'] = 'v1';
-	    headers['h2'] = 'v2';
-
-	    context.callback(new context.Response('response body', headers, 'text/plain', 201));
-	    return;
-	case 'return_fields':
-	    var fields = [];
-	    for (var key in event.fields) {
-		fields.push(key + '=' + event.fields[key]);
-	    }
-	    // We use sorted to get predictable output
-	    fields.sort();
-	    context.callback(fields.join(','));
-	    return;
-	case 'return_path':
-	    context.callback(event.path);
-	    return;
-	case 'return_error':
-	    throw 'some error';
-	default:
-	    throw 'Unknown return mode: ' + event.body;
+    switch (event.body.toString()) {
+        case 'return_string':
+            context.callback('a string')
+            return
+        case 'return_status_and_string':
+            context.callback([201, 'a string after status'])
+            return
+        case 'return_list':
+            context.callback([{a: 1}, {b: 2}])
```

Comment:
I wrote a comment here and I might have discarded it before sending it because I don't see it here anywhere anymore.
Anyway, it was about coding style of one-line object literals.
In most popular JS code-style guides, when you have an object literal on one line, you usually need to pad the braces `{` & `}` with a single space character:
```suggestion
            context.callback([{ a: 1 }, { b: 2 }])
```
This is in contrast to array brackets `[` & `]` that usually there is no such padding, so you **don't** do this:
```js
            context.callback([ { a: 1 }, { b: 2 } ])
```
You can let ESLint test this and fix this automatically for you.
The rule is named `object-curly-spacing` and you can read about it [here](https://eslint.org/docs/rules/object-curly-spacing).